BardMusicPlayer.Seer/Lib/Machina/TCPNetworkMonitor.cs

```diff
@@ -116,36 +116,20 @@ public void Stop()
         _connectionManager.Cleanup();
     }
 
-    private void ProcessDataLoop(CancellationToken token)
+    private async Task ProcessDataLoop(CancellationToken token)
     {
         try
         {
             while (!token.IsCancellationRequested)
             {
-                try
-                {
-                    _connectionManager.Refresh();
-
-                    ProcessNetworkData();
-
-                }
-                catch (OperationCanceledException)
-                {
-
-                }
-                catch (Exception ex)
-                {
-                    if (DateTime.UtcNow.Subtract(_lastLoopError).TotalSeconds > 5)
-                        Trace.WriteLine("TCPNetworkMonitor Error in ProcessDataLoop inner code: " + ex.ToString(), "DEBUG-MACHINA");
```

Comment:
This exception catch still needs to exist

Reply:
Re-added and doesn't seem to show up in profiler so all good. 👍